scrapy_time_machine/storages.py

```diff
@@ -129,7 +129,11 @@ def open_spider(self, spider):
     def _prepare_time_machine(self, settings):
         if settings.get("TIME_MACHINE_RETRIEVE"):
             # download db file from s3 snapshot_uri
-            compressed_db_file = self.s3_client.download_file('compressed_db_file', self.snapshot_uri)
+            s3bucket = self.s3.split("/")[2]
```

Comment:
You can also use urlparse to get the netloc without having to use split and indexes. 
```
>>> urlparse('s3://aws.bucket.com/nested/path').netloc
'aws.bucket.com'
```
In a similar way, you can use the `pathlib` to operate with the path and file suffixes: 

```
>>> p = pathlib.Path(urlparse('s3://aws.bucket.com/nested/path.gz').path)
>>> p.suffix
'.gz'
>>> p.parts
('/', 'nested', 'path.gz')
```